Microsoft/microsoft-sentinel/ingest/parser.yml

```diff
@@ -0,0 +1,32 @@
+name: microsoft-sentinel
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+  - name: set-common-fields
+  - name: set-sentinel-fields
+
+stages:
+  set-common-fields:
+    actions:
+      - set:
+          event.kind: "{{json_event.message.alertKind}}"
+          event.reason: "{{json_event.message.alertDescription}}"
+          observer.product: "{{ json_event.message.productName}}"
+          observer.vendor: "{{ json_event.message.vendorName}}"
+  set-sentinel-fields:
+    actions:
+      - set:
+          sentinel.alert.time: "{{json_event.message.alertTime}}"
```

Comment:
I think this field must be used in  `@timestamp` (use the stage date.parse in order to be sure to be compliant with our standard).